localstack/ext/java/src/main/java/cloud/localstack/docker/LocalstackDocker.java

```diff
@@ -0,0 +1,198 @@
+package cloud.localstack.docker;
+
+import cloud.localstack.LocalstackTestRunner;
+import cloud.localstack.ServiceName;
+import cloud.localstack.docker.command.RegexStream;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Logger;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+/**
+ * Localstack Docker instance
+ *
+ * @author fabianoo
```

Comment:
Since large portions of this file are copied from `LocalstackDockerTestRunner`, better also add the original author here:
```
@author Alan Bevier
@author fabianoo
```
Conversely, you may also want to add your name to the Java Doc of `LocalstackDockerTestRunner`.